domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/error/InvalidRouteTypeNotice.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2020. MobilityData IO.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.domain.entity.notice.error;
+
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.NoticeExporter;
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.base.ErrorNotice;
+
+import java.io.IOException;
+
+public class InvalidRouteTypeNotice extends ErrorNotice {
+
+    public InvalidRouteTypeNotice(final String filename, final String entityId) {
+        super(filename, E_026,
+                "Invalid Route type",
+                "Invalid Route type for Route with id:" + entityId + " in file:" + filename +
+                        ". A Route type should be an integer between 0 and 12.",
+                entityId);
+    }
+
+    @Override
+    public void export(final NoticeExporter exporter) throws IOException {
+        exporter.export(this);
+    }
+
+    public String getEntityId() {
```

Comment:
Can be removed
https://github.com/MobilityData/gtfs-validator/blob/12d9cf4e83726a06681cd56bcdd36e232ecfbdab/domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/base/Notice.java#L60